avm/res/network/network-manager/main.bicep

```diff
@@ -240,3 +238,158 @@ type roleAssignmentType = {
   @sys.description('Optional. The Resource Id of the delegated managed identity resource.')
   delegatedManagedIdentityResourceId: string?
 }[]?
+
+type networkGroupType = {
+  @sys.description('Required. The name of the network group.')
+  name: string
+
+  @sys.description('Optional. A description of the network group.')
+  description: string?
+
+  @sys.description('Optional. Static Members to create for the network group. Contains virtual networks to add to the network group.')
+  staticMembers: {
+    @sys.description('Required. The name of the static member.')
+    name: string
+
+    @sys.description('Required. Resource ID of the virtual network.')
+    resourceId: string
+  }[]?
+}[]?
+
+type networkManagerScopeAccessType = ('Connectivity' | 'SecurityAdmin')[]
```

Comment:
Just curious, is this the same as

```bicep
@allowed([
  'Connectivity'
  'SecurityAdmin'
])
param networkManagerScopeAccesses string[]
```
?

Reply:
Yes